dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
Or another (maybe not so great) option:
node = node.skolemize(self.curie_map.get(''))
node = re.sub(r'rdflib\/', '', node)
node = URIRef(node)